### DISCUSSION TITLE
How to disable the spell check in md file?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#2925] [Q&A] [answered]
Something like this I want to disable the spell check in markdown file.
I search in the doc of LazyVim get no result. So how to fix it?

![image](https://github.com/LazyVim/LazyVim/assets/11295355/34cd6edc-5162-40ac-9108-37f566adf1b3)

**Comment by dpetka2001** [comment, marked as answer]
`<leader>us`?? It's also mentioned in the [docs](https://www.lazyvim.org/keymaps#general).
